### DISCUSSION TITLE
Name conventions on generic diversity and PA targets

### DISCUSSION THREAD

**Discussion by David-OConnor** [ExpressLRS/ExpressLRS#1742] [Ideas]
Hi! I'm interested in submitting a PR to standardize the naming conventions of generic Rx targets: https://github.com/ExpressLRS/ExpressLRS/tree/master/src/hardware/RX.

Of note:
- [Generic 2400 Diversity](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/hardware/RX/Generic%202400%20Diversity.json) is full diversity, with PA

- [Generic 2400 PA Diversity](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/hardware/RX/Generic%202400%20PA%20Diversity.json) is antenna diversity, with PA

- [Generic 2400 Diversity](https://github.com/ExpressLRS/ExpressLRS/blob/master/src/hardware/RX/Generic%202400%20Diversity.json) is antenna diversity with no PA.

There are a number of ways this could be handled. One example is to label the first of those links `Generic 2400 PA Full Diversity`, while leaving the other 2 the same.

This is of note, since my understanding is targets are not being accepted for custom pinouts; following these designs is mandatory.

Also of note, is it possible to include comments in this? It's also unclear which MCUs these are for. Thank you.

**Comment by pkendall64** [comment]
Sure, a rename of the Full diversity target would be fine.

It's not that we are not accepting new target definitions, it's that we recommend using the ones we have to stop a proliferation of target files, also the ones we have are verified working by the devs.

This is especially true for users building DIY hardware; the whole idea of the unified targets with the json file is that you can just upload the json file to the device after flashing so theres actually no need to have the json pre-attached to the firmware.

**Reply by David-OConnor** [reply]
Thanks - I appreciate the info!
